commons-core/src/main/scala/com/avsystem/commons/serialization/legacyOptionEncoding.scala

```diff
@@ -0,0 +1,19 @@
+package com.avsystem.commons
+package serialization
+
+/**
+  * This ugly workaround has been introduced when standard `Option` encoding changed from zero-or-one element list
```

Comment:
Don't be so hard on yourself, seen worse than that.

Reply:
OK